### PR TITLE
NEXUS-24692: Fix broken api

### PR DESCRIPTION
**Pull request by aornatovskyy**
Ticket: https://issues.sonatype.org/browse/NEXUS-24692
CI: https://jenkins.ci.sonatype.dev/job/nxrm/job/nxrm3/job/nxrm-community-it/job/NEXUS-24681_nxrm_jenkins_shared_in_community_it/11/consoleFull

### CODE REVIEW

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/rest/ConanProxyRepositoriesApiResourceBeta.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+import org.sonatype.nexus.repository.rest.api.AbstractRepositoryApiRequestToConfigurationConverter;
+import org.sonatype.nexus.repository.rest.api.AuthorizingRepositoryManager;
+import org.sonatype.nexus.repository.rest.api.RepositoriesApiResourceBeta;
+import org.sonatype.repository.conan.api.ConanProxyRepositoryApiRequest;
+
+@Named
+@Singleton
+@Path(RepositoriesApiResourceBeta.RESOURCE_URI + "/conan/proxy")
+public class ConanProxyRepositoriesApiResourceBeta
```
Comment (mlukaretkyi) [inline nexus-repository-conan/src/main/java/org/sonatype/repository/conan/rest/ConanProxyRepositoriesApiResourceBeta.java:28]: Add Deprecated

Comment (mlukaretkyi) [reply nexus-repository-conan/src/main/java/org/sonatype/repository/conan/rest/ConanProxyRepositoriesApiResourceBeta.java:28]: @Api(hidden = true)

Comment (aornatovskyy) [reply nexus-repository-conan/src/main/java/org/sonatype/repository/conan/rest/ConanProxyRepositoriesApiResourceBeta.java:35]: done @mlukaretkyi

File: nexus-repository-conan/src/main/java/org/sonatype/repository/conan/rest/ConanProxyRepositoriesApiResourceBeta.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+import org.sonatype.nexus.repository.rest.api.AbstractRepositoryApiRequestToConfigurationConverter;
+import org.sonatype.nexus.repository.rest.api.AuthorizingRepositoryManager;
+import org.sonatype.nexus.repository.rest.api.RepositoriesApiResourceBeta;
+import org.sonatype.repository.conan.api.ConanProxyRepositoryApiRequest;
+
+import io.swagger.annotations.Api;
+
+/**
+ * @since 1.0.18
+ */
```
Comment (Mykyta) [inline nexus-repository-conan/src/main/java/org/sonatype/repository/conan/rest/ConanProxyRepositoriesApiResourceBeta.java:29]: For the classes with @Deprecated annotation add pls class header with Description:
```
@deprecated the 'beta' prefix is being phased out, prefer starting new APIs with {@link APIConstants#V1_API_PREFIX} instead. 
```

**Pull request merged**